.github/workflows/ci-a11y.yml

```diff
@@ -0,0 +1,46 @@
+name: Accessibility Tests
+
+on:
+  push:
+    branches:
+      - main
+      - next
```

Comment:
A11y tests now run on all pushes to `main` and `next`... worth pointing out and sense checking